fastpair/rust/src/bluetooth/windows_ble/adapter.rs

```diff
@@ -35,6 +73,87 @@ impl BleAdapter {
 
         Ok(BleAdapter { inner })
     }
+
+    pub fn scanner(&self) -> Result<impl Stream<Item = BleDevice>, anyhow::Error> {
+        let watcher = BluetoothLEAdvertisementWatcher::new()?;
+        watcher.SetScanningMode(BluetoothLEScanningMode::Active)?;
```

Comment:
this didn't work for me afaik, so maybe catching an error here and logging a warning instead may be a better approach.

Reply:
Unsure what you mean by this, which part didn't work for you and which warning should I be logging?

Reply:
the SetScanningMode to be specific, apparently it's a hardware-dependent call? weird but yeah definitely log a warning if that was unable to be set, but i should imagine the applet would still work.